hazelcast/src/main/java/com/hazelcast/jet/impl/operation/TerminateJobOperation.java

```diff
@@ -24,28 +24,36 @@
 import java.io.IOException;
 import java.util.concurrent.CompletableFuture;
 
+import static java.util.concurrent.CompletableFuture.completedFuture;
+
 /**
  * Operation sent from client to coordinator member to terminate particular
  * job. See also {@link TerminateExecutionOperation}, which is sent from
  * coordinator to members to terminate execution.
- * <p>
- * Not used for light jobs, see {@link CancelLightJobOperation}.
  */
 public class TerminateJobOperation extends AsyncJobOperation {
 
     private TerminationMode terminationMode;
+    private boolean isLightJob;
 
     public TerminateJobOperation() {
     }
 
-    public TerminateJobOperation(long jobId, TerminationMode mode) {
+    public TerminateJobOperation(long jobId, TerminationMode mode, boolean isLightJob) {
         super(jobId);
         this.terminationMode = mode;
+        this.isLightJob = isLightJob;
+        assert !isLightJob || terminationMode == TerminationMode.CANCEL_FORCEFUL;
```

Comment:
Could you add this comment on top of this assertion:
"Asserts that `isLightJob` => `terminationMode == TerminationMode.CANCEL_FORCEFUL`"